### PR TITLE
Create tutorial markdown

### PR DESCRIPTION
**Pull request by ykern**

### CODE REVIEW

**Comment by ykern** [comment]
#5

**Comment by ykern** [comment]
closes #4

File: tutorial.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Tutorial
+This is a tutorial to get started with the Cryosphere Virtual Laboratory (CVL).
+
+## What is CVL
+CVL stands for Cryosphere Virtual Laboratory. It offers users access to the following tools:
```
Comment (mitkin) [inline tutorial.md:5]: Maybe we could mention that it provides access to the datasets for Polar regions

File: tutorial.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+1. Data portal
+The data catalogue on the CVL websites allows users to browse metadata from various data publishers to locate data from multiple sources.
+2. Computing environment
+In collaboration with Polar Thematic Exploitation Platform (PTEP) a virtual computing environment (JupyterLab) is offered. This allows users to use virtual ressources instead of local machines.
+3. 3D Viewer
+The 3D Visualiser is a tool to illustrate user projects.
```
Comment (mitkin) [inline tutorial.md:11]: user generated data from multiple sources in the same view.

File: tutorial.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
+# Tutorial
+This is a tutorial to get started with the Cryosphere Virtual Laboratory (CVL).
+
+## What is CVL
+CVL stands for Cryosphere Virtual Laboratory. It offers users access to the following tools:
+1. Data portal
+The data catalogue on the CVL websites allows users to browse metadata from various data publishers to locate data from multiple sources.
+2. Computing environment
+In collaboration with Polar Thematic Exploitation Platform (PTEP) a virtual computing environment (JupyterLab) is offered. This allows users to use virtual ressources instead of local machines.
```
Comment (mitkin) [inline tutorial.md:9]: type "resources"

File: tutorial.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+[TODO diagram]
+
+## How to get and set up required accounts 
+Accounts for GitHub, PTEP and CVL are required to follow the workflow.
+
+### GitHub
+1. Log in to https://github.com/ (or register as new user)
+2. Navigate to https://github.com/CryosphereVirtualLab/public-notebooks
+3. Fork the repository, this will create a linked private repository at the following path (where YOUR-NAME is replaced by your username): https://github.com/YOUR-NAME/public-notebooks
```
Comment (mitkin) [inline tutorial.md:24]: A fork is user copy of the original repository. Once the user is happy with their changes, a pull request could be submitted to the original repository.

File: tutorial.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+            
+- Clone the repository from GitHub you forked earlier (replace YOUR-NAME with correct link):
+
+            git clone https://github.com/YOUR-NAME/public-notebooks
+            
+- Navigate to the cloned directory:
+
+            cd public-notebooks
+            
+- The following will update the work environment with missing python packages (confirm with `y` once asked):
```
Comment (mitkin) [inline tutorial.md:45]: The following command

File: tutorial.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+            source provision.sh
+    
+
+### CVL
+1. Register at: https://cvl.eo.esa.int/user/login
```
Comment (mitkin) [inline tutorial.md:51]: "Once registration is submitted, portal administrator will have to approve it"

File: tutorial.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+
+## How to find source data
+1. Got to the CVL website: https://cvl.eo.esa.int/
+2. Navigate to `Data`
+3. Use the catalogue to browse through metadata
```
Comment (mitkin) [inline tutorial.md:57]: 4. Once you find the required datasets, click "Add to basket" if that options is provided. In some cases you will only obtain link to the data, or link to the dataset landing page at a different resource. In that situation, you will need to note down the link and obtain data yourself in the computing environment, e.g. using `wget` or `requests` Python package.

File: tutorial.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+1. Login to https://polartep.io/jupyter/hub/home
+2. In the left sidepanel open the file browser (Ctrl+Shift+F)
+3. Navigate to the desired directory
+4. Press the blue `+` button on the top. This opens a Launcher tab. Here click the `Notebook` panel.
+
+To get started with Jupyter Notebook check out the `cvl_get-started.ipynb` in the `public-notebooks` directory you cloned earlier from GitHub.
+
+
+
+## Submitting to public notebooks
```
Comment (mitkin) [inline tutorial.md:73]: Submitting a pull request to the `public-notebooks` repository

File: tutorial.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+
+[TODO]
```
Comment (mitkin) [inline tutorial.md:75]: Add a link to the official documentation for "how to create a personal access token"

**Comment by ykern** [comment]
@mitkin pushed the new version of the Jupyter Notebook.
If the OPEnDAP and non-OPEnDAP part is fine then we would only need to add a raster to the map.
That would be the last part missing to our original plan on what to show in that file.

One issue seems to be during the first import of libraries. It raises an error as you can see but the notebook still seems to work for the rest.

**Pull request merged**